api-report/test-runtime-utils.api.md

```diff
@@ -142,6 +144,20 @@ export class MockContainerRuntimeFactory {
     sequenceNumber: number;
 }
 
+// @public
+export class MockContainerRuntimeFactoryForRebasing extends MockContainerRuntimeFactory {
+    // (undocumented)
+    createContainerRuntime(dataStoreRuntime: MockFluidDataStoreRuntime, overrides?: {
+        minimumSequenceNumber?: number;
```

Comment:
While I can see why you might want to create shape like that for last arg, I'd rather switch to this form when it's needed. Optional inside of optional is not great design pattern :)